src/main/java/bro.java

```diff
@@ -0,0 +1,119 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class bro {
+
+    private static final ArrayList<String> storer = new ArrayList<>();
+    private static final ArrayList<String> mark_tracker = new ArrayList<>();
+
+    public static void level0() {
+        System.out.println("Hello! I'm bro");
+        System.out.println("What can I do for you?");
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    public static void echo() {
+
+        String line;
+        Scanner in = new Scanner(System.in);
+        while (true) {
+            line = in.nextLine();
+
+            if (line.equals("Bye")) {
+                break;
+            }
+            System.out.println(line);
+        }
+
+        System.out.println("Bye. Hope to see you again soon!");
+
+    }
+
+    public static void addList() {
+
+        String line;
+        Scanner in = new Scanner(System.in);
+
+        while (true) {
```

Comment:
Wait if you have while(true) loop in your addList and you call your mark only after addList, wouldnt you be marking after the user inputs bye.